Use only country name for tooltip

Bluesmile82 opened the pull request:
There is no design for the tooltip but it was decided it should be only the country name.

![image](https://user-images.githubusercontent.com/9701591/69366333-6ce81c00-0c96-11ea-92f5-67b6461aacf9.png)

kfilip94 commented:
On desktop, tooltip looks and works nicely:+1: There is a typical problem with ReactTooltip library on mobiles - after clicking the tooltip doesn't disappear, so the best would be to disable it for mobiles.

Also, there is another glitch on mobiles: share buttons and search input overlap, I know that this is not connected with this PR but I shouldn't be hard to fix.
![puwsw-whcl7](https://user-images.githubusercontent.com/15097138/69619787-0e26f780-1034-11ea-9aec-dceb7b05570d.gif)

After that it'll be ready to merge ;)

Bluesmile82 commented:
Thanks! For the mobile problem, we have a delayHide attribute that was in 3000 ms. I changed it to 1000. I also fixed the styles

![image](https://user-images.githubusercontent.com/9701591/69643021-f450d900-1062-11ea-81c3-2bae5a59e025.png)
